fuzzy/verifier.go

```diff
@@ -45,7 +45,13 @@ func (v *appendEntriesVerifier) PreRequestVote(src, target string, rv *raft.Requ
 
 func (v *appendEntriesVerifier) PreAppendEntries(src, target string, req *raft.AppendEntriesRequest) (*raft.AppendEntriesResponse, error) {
 	term := req.Term
-	ldr := string(req.Leader)
+	var ldr string
+	if len(req.RPCHeader.Addr) > 0 {
+		ldr = string(req.RPCHeader.Addr)
+	} else {
+		ldr = string(req.Leader)
+	}
```

Comment:
Do we still need to do this? Any reason not to continue to use `req.Leader`? Why do we prefer the new `RPCHeader.Addr` ?

If we are trying to phase it out, we should document that on the `Leader` field.

Same question about the other two places we look at `Leader` or `Candidate`.

Reply:
I think we should aim to phaseout Leader and Candidate parameters, it make more sense to infer those values from the RPCHeader now and removing those will reduce the on the wire packet size sensibly. Also if we keep them it mean we need to document/test special cases behaviour (different values, Addr empty and Candidate not...). 
I will add the deprecation comment for both Candidate and Leader